### PR TITLE
faq: Describe in what condition TiDB and RawKV can be used together

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Describe in what condition TiDB & RawKV can be used together.

Since `v6.1.0` TiDB & RawKV can be used together if [API V2](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#api-version-new-in-v610)  is enabled:
> Data is scoped according to different usage and API V2 supports co-existence of TiDB, Transactional KV, and RawKV applications in a single cluster.

But the relevant FAQ has not been changed accordingly yet.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/17828
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: faq/manage-cluster-faq.md [hunk 1]
```diff
@@ -378,7 +378,10 @@ TiKV 的内存占用主要来自于 RocksDB 的 block-cache，默认为系统总
 
 ### TiDB 数据和 RawKV 数据可存储于同一个 TiKV 集群里吗？
 
-不可以。TiDB 数据（或使用其他事务 API 生成的数据）依赖于一种特殊的键值格式，和 RawKV API 数据（或其他基于 RawKV 的服务生成的数据）并不兼容。
+这取决于你使用的 TiDB 版本以及是否启用了 TiKV API V2 （[`storage.api-version = 2`](/tikv-configuration-file.md#api-version-new-in-v610)）。
```
Comment (qiancai) [inline faq/manage-cluster-faq.md:381]:
```suggestion
这取决于你使用的 TiDB 版本以及是否启用了 TiKV API V2 （即 [`storage.api-version = 2`](/tikv-configuration-file.md#api-version-从-v610-版本开始引入)）。
```

**Review by pingyu — APPROVED** [review]
LGTM~

File: faq/manage-cluster-faq.md [hunk 2]
```diff
@@ -378,7 +378,10 @@ TiKV 的内存占用主要来自于 RocksDB 的 block-cache，默认为系统总
 
 ### TiDB 数据和 RawKV 数据可存储于同一个 TiKV 集群里吗？
 
-不可以。TiDB 数据（或使用其他事务 API 生成的数据）依赖于一种特殊的键值格式，和 RawKV API 数据（或其他基于 RawKV 的服务生成的数据）并不兼容。
+这取决于你使用的 TiDB 版本以及是否启用了 TiKV API V2 （即 [`storage.api-version = 2`](/tikv-configuration-file.md#api-version-从-v610-版本开始引入)）。
+
+- 如果你使用的是 v6.1.0 或之后版本的 TiDB，并且启用了 TiKV API V2，那么 TiDB 数据和 RawKV 数据可以共存于同一个 TiKV 集群。
+- 否则，不可以将 TiDB 数据和 RawKV 数据存储于同一个 TiKV 集群中，因为 TiDB 的 key 的格式（或使用事务 API 创建的数据）与使用 RawKV API 创建的数据（或来自其他基于 RawKV 的服务生成的数据）不兼容。
```
Comment (qiancai) [inline faq/manage-cluster-faq.md:384]:
```suggestion
- 否则，不可以将 TiDB 数据和 RawKV 数据存储于同一个 TiKV 集群中，因为 TiDB 数据的 key 的格式（或使用事务 API 创建的数据）与使用 RawKV API 创建的数据（或来自其他基于 RawKV 的服务生成的数据）不兼容。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**